app/controllers/reptiles_controller.rb

```diff
@@ -21,6 +21,9 @@ def update
     end
 
     def destroy
+        reptile = Reptile.find(params[:id])
+        reptile.destroy
+        render json: reptile
```

Comment:
Are you sure you want to render the data entry that was just removed? Is there some other way that you can show that the entry was deleted?

Reply:
We changed it so it renders Reptile.all instead.